src/main/scripts/org/reaktivity/specification/nukleus/http/streams/rfc7230/architecture/request.and.response/raw/server.rpt

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2016-2017 The Reaktivity Project
+#
+# The Reaktivity Project licenses this file to you under the Apache License,
+# version 2.0 (the "License"); you may not use this file except in compliance
+# with the License. You may obtain a copy of the License at:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+
+# source (cooked) -> http -> target (this script) -> network
+# Read from target/http, write to http/target
+
+property source "http"
+property newTargetOutputRef ${nuklei:newReferenceId()} # external scope
+       
+accept "http://localhost:8080/" option http:transport "nukleus://streams/target"
+       option nukleus:source ${source}
+       option nukleus:route ${newTargetOutputRef}
+       option nukleus:window 8192
+       option nukleus:transmission "duplex"
```

Comment:
Good point.

I think it might be better to generally handle `options` rather than special-casing the concept of transport.

For example:
```
property httpOptions { "http:transport":"nukleus://streams/target", "nukleus:source": ... }
accept "http://localhost:8080/" options ${httpOptions}
```

Actually, with that in place, it might be better to write the original one as follows:

```
property httpOptions {} // empty

accept "http://localhost:8080/" options ${httpOptions}
```

then use a property override to retarget the transport and options all at once.